pkgs/development/ocaml-modules/dune-configurator/default.nix

```
@@ -1,17 +1,23 @@
-{ lib, buildDunePackage, dune_2, csexp, result }:
+{ lib, buildDunePackage, dune_3, csexp }:
 
 buildDunePackage rec {
   pname = "dune-configurator";
 
-  inherit (dune_2) src version patches;
+  inherit (dune_3) src version patches;
 
-  duneVersion = "2";
+  # This fixes finding csexp
+  postPatch = ''
+    rm -rf ./vendor
+    rm -rf vendor/pp vendor/csexp
```

```suggestion
    rm -rf ./vendor
    rm -rf vendor/pp vendor/csexp
```

Isn't it removing the same directory that was already removed in previous line?

---

Bad rebase, mine 10 should be dropped